tests/jeos/firstrun.pm

```diff
@@ -132,8 +132,8 @@ sub verify_partition_label {
 
 sub verify_selinux {
     if (has_selinux_by_default) {
-        # SELinux is default, should be enabled and in enforcing mode
-        validate_script_output('sestatus', sub { m/SELinux status: .*enabled/ && m/Current mode: .*enforcing/ }, fail_message => 'SELinux is NOT enabled and set to enforcing');
+        my $mode = is_sle('>=16.0') && is_sles4sap() ? 'permissive' : 'enforcing';
+        validate_script_output('sestatus', sub { m/SELinux status: .*enabled/ && m/Current mode: .*$mode/ }, fail_message => "SELinux is NOT enabled and set to $mode");
```

Comment:
I would also change the `fail_message` because SELinux could be enabled but not set to the desired mode.